CHANGELOG.md

```diff
@@ -49,10 +60,15 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 - Agent hanging if plugins do not shut down. #3557
 - WMI exploiter hanging. #3543
 - Discovered network services are displayed in reports. #3000
+- Windows ransomware README line endings. #3699
+- Services count in scanned servers table in security report. #3708
+- Windows manual run command. #3709
+- Spacing in encrypted files table in ransomware report. #3715
```

Comment:
I believe the 1st, 3rd, and 4th items in this list are things we broke during the release and subsequently fixed. In other words, I don't think these are bugs in 2.2.0 that are fixed by 2.3.0. As such, they don't belong in the changelog. #2 is a good catch, though.

Feel free to double check me on that.

Reply:
I think you're right, removed 1st, 3rd, 4th